client/blocks/like-button/README.md

```diff
@@ -8,7 +8,7 @@ For most usage, the container is the easiest route.
 #### How to use the container:
 
 ```js
-var LikeButtonContainer = require( 'components/like-button' );
+var LikeButtonContainer = require( 'blocks/like-button' );
```

Comment:
Optional, but it would be nice to update the usage examples to our current code guidelines